github/repos_deployments.go

```diff
@@ -134,16 +134,21 @@ func (s *RepositoriesService) CreateDeployment(ctx context.Context, owner, repo
 type DeploymentStatus struct {
 	ID *int64 `json:"id,omitempty"`
 	// State is the deployment state.
-	// Possible values are: "pending", "success", "failure", "error", "inactive".
-	State         *string    `json:"state,omitempty"`
-	Creator       *User      `json:"creator,omitempty"`
-	Description   *string    `json:"description,omitempty"`
-	TargetURL     *string    `json:"target_url,omitempty"`
-	CreatedAt     *Timestamp `json:"created_at,omitempty"`
-	UpdatedAt     *Timestamp `json:"updated_at,omitempty"`
-	DeploymentURL *string    `json:"deployment_url,omitempty"`
-	RepositoryURL *string    `json:"repository_url,omitempty"`
-	NodeID        *string    `json:"node_id,omitempty"`
+	// Possible values are: "pending", "success", "failure", "error",
+	// "inactive", "in_progress", "queued".
+	State          *string    `json:"state,omitempty"`
+	Creator        *User      `json:"creator,omitempty"`
+	Description    *string    `json:"description,omitempty"`
+	TargetURL      *string    `json:"target_url,omitempty"`
+	CreatedAt      *Timestamp `json:"created_at,omitempty"`
+	UpdatedAt      *Timestamp `json:"updated_at,omitempty"`
+	DeploymentURL  *string    `json:"deployment_url,omitempty"`
+	RepositoryURL  *string    `json:"repository_url,omitempty"`
+	EnvironmentURL *string    `json:"environment_url,omitempty"`
+	LogURL         *string    `json:"log_url,omitempty"`
+	NodeID         *string    `json:"node_id,omitempty"`
+	URL            *string    `json:"url,omitempty"`
+	Environment    *string    `json:"environment,omitempty"`
```

Comment:
Would you mind moving `Environment` up immediately after `Description`, followed by `NodeID`?
That way, all the URLs are together, and Creator, Description, Environment, and NodeID are at least in sorted order.

Reply:
Done. I moved `TargetURL` down so it's grouped with the other URLs as well.